topology/University of Colorado - Boulder/Research Computing/CUmulus.yaml

```
@@ -0,0 +1,1366 @@
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID: <ID>
+
+# Resources contains one or more resources in this
+# ResourceGroup. A resource provides one or more services
+Resources:
+  # Resource Name should be a short descriptor of the resource.
+  # e.g. the Center for High Throughput Computing's GlideinWMS Frontend is "CHTC-glidein2"
+  # Resource Names need to be unique across all resources in the OSG.
+  porter1s1:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: false
+    # Description is a long description of the resource; may be multiple lines
+    Description: This is a Research Computing CUmulus cloud host for the University of Colorado - Boulder.
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID: <ID>
+    # ContactLists contain information about people to contact regarding this resource.
+    # The "ID" is a hash of their email address available at https://topology.opensciencegrid.org/miscuser/xml
+    # If you cannot find the contact above XML, please register the contact:
+    # https://opensciencegrid.org/docs/common/registration/#registering-contacts
+    ContactLists:
+      # Administrative Contact is one to three people to contact regarding administrative issues
+      Administrative Contact:
+        Primary:
+          Name: Brett Shouse
+          ID: 25da6f6a72dfe6fc8b25da208a7c69d81171f29e
+        # Secondary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+      # Security Contact is one to three people to contact regarding security issues
+      Security Contact:
+        Primary:
+          Name: Brett Shouse
+          ID: 25da6f6a72dfe6fc8b25da208a7c69d81171f29e
+        # Secondary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+    # FQDN is the fully qualified domain name of the host running this resource
+    FQDN: porter1s1.rc.int.colorado.edu
+    ### FQDNAliases (optional) are any other DNS aliases by which this host can be accessed
+    # FQDNAliases:
+    #   - <HOSTNAME1>
+    #   - <HOSTNAME2>
+
+    ### DN (optional except for XCache resources) is the DN of the host cert of the resource
+    # in OpenSSL 1.0 format (i.e. /DC=org/DC=incommon/C=US/...)
+    # DN: <DN>
+
+    # Services is one or more services provided by this resource;
+    # valid services are listed in topology/services.yaml with the format "<SERVICE NAME>: <ID>"
+    Services:
+      Virtual Machine Host:
```

```suggestion
      CE:
```

We don't yet have a dedicated service type for the backfill containers so we're going to use "CE" to make sure that the accounting shows up properly